docs/docs/reference/04-nextjs/index.md

```diff
@@ -8,11 +8,11 @@ title: Next.js
 This feature is experimental and may be removed or changed in the future.
```

Comment:
Since we don't have Next.js Auth yet (`@auth/nextjs`), this page might be a bit confusing. The docs is still for NextAuth.js (`next-auth`). We should add a warning at the top saying this is going to be for Next.js Auth, and current docs is at https://next-auth.js.org